test/fips_version_test.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2020-2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+
+ * These tests are setup to load null into the default library context.
+ * Any tests are expected to use the created 'libctx' to find algorithms.
```

Comment:
Maybe this comment should be updated?

Reply:
I think this is what it is doing. @t8m?

Reply:
This was copied from something that did multiple tests. This just performs the one utility function.

Reply:
If this is supposed to test version for any provider then it need to be renamed (and the file renamed). If not, it should be clear what it does.

Reply:
It also doesn't seem to work:
```sh
$ ./util/wrap.pl test/fips_version_test '=3.0.0' '!3.0.0'
Useless use of a variable in void context at ./util/wrap.pl line 63.
1..1
    # Subtest: test_fips_version
    1..2
    ok 1 - iteration 1
    ok 2 - iteration 2
ok 1 - test_fips_version
```
Can't both be 3.0.0 and not 3.0.0.